src/lib/blueprint.ts

```diff
@@ -299,7 +300,42 @@ const createRoutes = async (
   return Array.from(routeMap.values())
 }
 
+const getNamespace = (path: string, paths: OpenapiPaths): string | null => {
+  // Hold namespace in array to allow nested namespaces
+  // e.g. namespace for `/foo/bar/baz/get` = `/foo/bar`
+  const namespace: string[] = []
+
+  const pathParts = path.split('/').filter((path) => Boolean(path))
+
+  const pathKeys = Object.keys(paths)
+
+  for (const [index, part] of pathParts.entries()) {
+    // Namespaces must be consecutive. If there was a path with an endpoint
+    // previously, then this part is not in the namespace.
+    if (namespace.length !== index) {
+      continue
+    }
+
+    // An endpoint is a route that ends without further paths. i.e., ends in
+    // a letter (not slash).
+    const endpoints = pathKeys.filter((key) =>
+      new RegExp(`^/${[...namespace, part].join('/')}/\\w+$`).test(key),
+    )
```

Comment:
Do we need the regex, or can we just do `!endsWith('/')`?

Reply:
lol, that works too 😂. Thanks!

Reply:
oops, this regex ended up being required because we want to test the rest of the parts as well.

Reverted in https://github.com/seamapi/blueprint/pull/83/commits/7cc9bc7ef0abdfc9c1a46e9e43c8ae525a83baaa